third_party/2/google/protobuf/internal/well_known_types.pyi

```diff
@@ -0,0 +1,91 @@
+from typing import Any, Optional
+
+class Error(Exception): ...
+class ParseError(Error): ...
+
+# This is named 'Any' in the original, but that conflicts with typing.Any,
```

Comment:
Why don't we `from typing import Any as Any_` instead to avoid this?

Reply:
Because that would require more changes to the code below (generated by stubgen), all of which uses `Any` as meaning `typing.Any`.